### PR TITLE
Translated first page of exported article

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes I think 😇 
| Documentation | no
| Translation   | yes
| Fixed tickets | #3277
| License       | MIT

### CODE REVIEW

File: src/Wallabag/CoreBundle/Helper/EntriesExport.php [hunk 1]
```diff
@@ -30,10 +32,11 @@ class EntriesExport
      * @param string $wallabagUrl Wallabag instance url
      * @param string $logoPath    Path to the logo FROM THE BUNDLE SCOPE
      */
-    public function __construct($wallabagUrl, $logoPath)
+    public function __construct($wallabagUrl, $logoPath, TranslatorInterface $translator)
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Helper/EntriesExport.php]: I would have put $translator as first argument, but it's okay.

**Comment by nicosomb** [comment]
Unrelated changed are due to php-cs.

**Comment by Strubbl** [comment]
Hey Julian @jlnostr , wie findest du die Übersetzung? https://github.com/wallabag/wallabag/pull/3372/files#diff-1b88e4fea007282503aba471bd99e6d4

File: src/Wallabag/CoreBundle/Helper/EntriesExport.php [hunk 2]
```diff
@@ -17,21 +18,19 @@ class EntriesExport
...
     private $author = 'wallabag';
     private $language = '';
-    private $footerTemplate = '<div style="text-align:center;">
-        <p>Produced by wallabag with %EXPORT_METHOD%</p>
-        <p>Please open <a href="https://github.com/wallabag/wallabag/issues">an issue</a> if you have trouble with the display of this E-Book on your device.</p>
-        </div>';
 
     /**
      * @param string $wallabagUrl Wallabag instance url
      * @param string $logoPath    Path to the logo FROM THE BUNDLE SCOPE
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Helper/EntriesExport.php]: Need to update the phpdoc

Comment (nicosomb) [reply src/Wallabag/CoreBundle/Helper/EntriesExport.php]: Fixed.

**Comment by Strubbl** [comment]
@nicosomb Does this also fix https://github.com/wallabag/wallabag/issues/3276?

**Comment by nicosomb** [comment]
I don't think.

**Pull request merged**